### PR TITLE
Implement a dev-room container to simulate a real room locally

### PR DESCRIPTION
**Pull request by lftakakura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,11 +134,20 @@ To run on dev:
...
+
+3) Start dependencies
 ```
 make deps
 make drop
-make migration
+make migrate
 ```
 
 3) Start worker with `make work` and api with `make run-dev`
```
Comment (Henrod) [inline README.md]: you forgot to increment the next ones hehe

File: models/room_address.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		for _, port := range roomSvc.Spec.Ports {
+			if port.NodePort != 0 {
+				rAddresses.Ports = append(rAddresses.Ports, &RoomPort{
+					Name: port.Name,
+					Port: port.NodePort,
+				})
+			}
+		}
+	} else {
+		for _, port := range roomPod.Spec.Containers[0].Ports {
```
Comment (Henrod) [inline models/room_address.go]: You should get the port from all containers

File: models/room_address.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+				rAddresses.Ports = append(rAddresses.Ports, &RoomPort{
+					Name: port.Name,
+					Port: port.NodePort,
+				})
+			}
+		}
+	} else {
+		for _, container := range roomPod.Spec.Containers {
+			for _, port := range container.Ports {
+				//TODO: check if port.HostPort is available (another process not using it)
```
Comment (Henrod) [inline models/room_address.go]: This TODO does not make sense anymore, you can remove it

**Comment by lftakakura** [comment]
Will create another PR to eliminate conflicts

**Pull request closed without merging**